### PR TITLE
Feature 0.0.2

### PR DESCRIPTION
**Pull request by himanshiparnami**

### CODE REVIEW

File: tasks/consul_user.yml [hunk 1]
```diff
@@ -4,8 +4,9 @@
     state: present
```
Comment (b44rawat) [inline tasks/consul_user.yml:4]: Do we need this part here?

**Comment by b44rawat** [comment]
does it provide both cluster & standalone setup?

File: tasks/consul_user.yml (hunk 1 above)
Comment (himanshiparnami) [reply tasks/consul_user.yml:4]: No by default the state will be present

**Comment by himanshiparnami** [comment]
> does it provide both cluster & standalone setup?

yes

**Pull request merged**